docs/video-miners/getting-started/testing/testnet.md

```diff
@@ -3,38 +3,40 @@ title: Running on Testnet
 sidebar_position: 2
 ---
 
-This guide will walk you through the process of running an orchestrator on the Rinkeby testnet. This is an important step to (1) ensure that your orchestrator is correctly configured, and (2) take any troubleshooting steps and make any tweaks in a lower-risk environment than mainnet.
+This guide will walk you through the process of running an orchestrator on the Arbitrum Rinkeby testnet. This is an important step to (1) ensure that your orchestrator is correctly configured, and (2) take any troubleshooting steps and make any tweaks in a lower-risk environment than mainnet.
 
-1. **Acquire some Rinkeby ETH from a faucet**
+1. **Acquire some Rinkeby ETH from a faucet and bridge it to Arbitrum**
 
-You can use an existing third-party faucet for this. You will want to send the ETH to the wallet that you will be using to run your orchestrator. We recommend [the official Rinkeby faucet](https://www.rinkeby.io/#faucet) (requires Twitter or Facebook auth), or [the Chainlink faucet](https://faucets.chain.link/rinkeby) (no auth required). Most faucets provide ~0.1 ETH per request, but you can use them multiple times if you need more.
+You can use an existing third-party faucet for this. We recommend [the official Rinkeby faucet](https://www.rinkeby.io/#faucet) (requires Twitter or Facebook auth), or [the Chainlink faucet](https://faucets.chain.link/rinkeby) (no auth required). Most faucets provide ~0.1 ETH per request, but you can use them multiple times if you need more.
 
-2. **Get a Rinkeby RPC url, either by running a node or by using a url from Infura or Alchemy**
+Once you've acquired the Rinkeby ETH, use the [Arbitrum bridge](https://bridge.arbitrum.io) to send Rinkeby ETH to Arbitrum. After bridging, you will want to send the Arbitrum Rinkeby ETH to the wallet that you will be using to run your orchestrator.
 
-To run your own `geth` node, please follow [Rinkeby’s official guide](https://www.rinkeby.io/#geth). If you’d prefer to use an [Infura](https://infura.io/) or [Alchemy](https://www.alchemy.com/) RPC url, please follow the guides on their respective sites.
+2. **Get an Arbitrum Rinkeby RPC url, either by running a node or by using a url from Infura or Alchemy**
```

Comment:
```suggestion
2. **Get an Arbitrum Rinkeby RPC url**
```

Removed the options here, since I think the first recommendation is to simply use the official testnet endpoints